04LessonFour/in-class-project/index.html

```diff
@@ -1,61 +1,72 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <title>The Daily Planet</title>
-  <link rel="stylesheet" href="css/style.css">
-</head>
-
-<body>
-
-  <div id="head">
-
-    <h1>Intel</h1>
-
-    <nav>
-      <a href="http://www.google.com">i7 Processor</a>
-      <a href="http://www.google.com">Solid State Drives</a>
-    </nav>
-
-  </div>
-
-<!--Body 1-->
-  <div id="body1">
-
-    <div>
-      <h3>Intel(R)Core<sup>TM</sup>i7</h3>
-      <h6>Hey hey hey hey hey hey hey hello hey hey hey hey. Hey hey hey hey hey, hey hey hello. Hello hey hey hello.</h6>
-    </div>
-
-    <img src="img/processor.jpg">
-<!--Body 1 end-->
-
-<!--"Ready Wake Create" Section-->
-    <div id="ready">
-      <h3>Ready. Wake. Create.</h3>
-      <h6>More words here, boop boop boop boop boop boop boop boop.</h6>
-    </div>
-
-  </div>
-<!--"Ready Wake Create" Section End-->
-
-<!--Body2-->
-    <div id="body2">
-      <h3>Intel Customer Support</h3>
-      <h6>Words Words Words, hello hey hi hey hello. hey hi hello hello hullo hi hello hey <a href="http://www.google.com">support</a></h6>
-    </div>
-<!--Body2 End-->
-
-<!--Body3-->
-    <div id="body3">
-      <h1>Overview: Intel(R)Solid State Drives</h1>
-      <img src="img/ssd.jpg" />
-        <div id="arch">
-          <h4>Architect of the Future</h4>
-          <h6>Words words words and words words words hey hey hey hey hello hey hi hello hey.</h6>
-        </div>
-    </div>
-<!--Body3 End-->
-
-  </div>
-</body>
-</html>
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>The Daily Planet</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <!--Header Start-->
+    <div id="head">
+
```

Comment:
I. Love. Your. CODE. so organized, commented, spaced, and legible! keep this up plz!